benchmarks/common.py

```diff
@@ -1,18 +1,28 @@
 import os
 import timeit
+"""
+if environment variable $MULTICORE is set, benchmarks will be run for
+n_jobs vals = (1, $MULTICORE). Otherwise for n_jobs vals = (1, -1)
+MULTICORE can be a list of values as well.
+Example: export MULTICORE=2,3,4,5
+"""
```

Comment:
It still seems not enough for me. How to run using specific number of threads without running single thread as well? Another use case: if I want to run on all the whole range up to HW processors available, should I know the number of processors and supply the whole list externally or probably better to introduce a notion of a range e.g. `MULTICORE=1:-1` would mean to run on all the numbers from 1 to all (e.g. 1,2,3,4,5,6,7,8 if -1 translates to 8)?

Reply:
let's fix the first part with `(1,$MULTICORE)` first for this PR and return to the range notion later

Reply:
Fixed